sw/device/silicon_creator/rom_ext/doc/rescue.md

```diff
@@ -79,7 +82,7 @@ ok: send boot_svc request via xmodem-crc
 ```
 
 The ROM_EXT will then will prompt for the transfer to start by sending the Xmodem-CRC start character (which is the ASCII character `C`).
-After completing this action, the ROM_EXT will switch back to firmware rescue mode.
```

Comment:
Was this previous comment accurate?  If so, why are you reversing the default behavior?  (As opposed to keeping the convention that after firmware transfer, the chip goes back to rescue mode, with a "RESET"/"BOOT" command that can be sent afterwards.)

Reply:
I see.  I find it a little cleaner to have RESQ never reboot, and a separate "BOOT" command, which would by default be issued by opentitantool.  But this maintains backwards compatibility, and gets the job done.